index.html

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0039)https://skycocker.github.io/chromebrew/ -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
It would be better to put the `<head>...` on a new line.

Reply:
done